### PR TITLE
fix to issue #35

### PR DESCRIPTION
**Pull request by mtuck063**
Did you delete your migration? I did not get any of them, and they are not up in the repo anymore. Our schema.rb are different, should I make all of them myself, or just forget about it?

### CODE REVIEW

**Comment by CraigBryan** [comment]
I think I've figured out the schema problem. What I didn't realize about schema is that it actually reflects the current state of the running database. So my dev database has all those extras, because I'm working on that stuff. So my schema.rb looks like that even if I delete it and run db:migrate.

I'll have to figure out a good way to undo those changes. It's probably best to bring my migrations over from my other branches and simply leave them there. Then if we change anything, we can add more migrations, and everything will still be A-OK.

In short, just leave your stuff as is. It's all good.

**Comment by mtuck063** [comment]
Alright.

**Comment by CraigBryan** [comment]
The issue wasn't actually fixed on my system. You didn't go far enough.

When I pushed it to 1200px, nothing wrapped to the next line (at 1100 it did, for me).

**Comment by mtuck063** [comment]
Whatever works, 1100px worked fine on safari. Better safe than sorry; just change it to 1200px.

**Comment by mtuck063** [comment]
Are you going to change the css files also? (application and header)

**Comment by CraigBryan** [comment]
Yup! I was doing it with the github online editor, and it only lets you do one file at a time.

**Comment by mtuck063** [comment]
Ah I see.

**Comment by katdrobnjakovic** [comment]
Awesome! Thanks guys! :)

**Comment by charle692** [comment]
This still doesn't seem to be fully fixed at least on my view here. Once I go below a width of 1108px I get this. 
image

It's a little hard to see but tips drops down. And stays like this until the mobile menu is activated.

**Comment by mtuck063** [comment]
I don't think he pushed the changes to heroku. Cause I just tested it and it doesnt go mobile till 1025px still.

**Comment by CraigBryan** [comment]
Correct. I want to get Arash's picture sorted before I push live.

**Comment by charle692** [comment]
ok sounds good was just checking.

**Comment by CraigBryan** [comment]
Is Arash on the slack? I can't find him there.

**Comment by charle692** [comment]
I remember he was at one point but doesn't seem like it anymore.
